pkg/networkmanager/interfaces.js

```diff
@@ -2567,26 +2837,29 @@ PageNetworkInterface.prototype = {
         var is_deletable = (iface && !dev) || (dev && (dev.DeviceType == 'bond' ||
                                                        dev.DeviceType == 'team' ||
                                                        dev.DeviceType == 'vlan' ||
-                                                       dev.DeviceType == 'bridge'));
+                                                       dev.DeviceType == 'bridge' ||
+                                                       dev.DeviceType == 'wifi'));
         $('#network-interface-delete').prop('hidden', !is_deletable || !managed);
 
         function render_carrier_status_row() {
             if (dev && dev.Carrier !== undefined) {
-                return $('<tr>').append(
-                    $('<td>').text(_("Carrier")),
-                    $('<td>').append(
-                        dev.Carrier
-                            ? (dev.Speed ? cockpit.format_bits_per_sec(dev.Speed * 1e6) : _("Yes"))
-                            : _("No")));
+                return `<tr>
```

Comment:
This change seems unmotivated to me.  IMO it's fine to keep using jQuery here since the rest of the code does it as well. There is no point in introducing two styles.